config/pipeline.yaml

```diff
@@ -96,3 +97,9 @@ device_types:
     class: shell
     params:
       shebang: '/usr/bin/env python3'
+
+
+email_configs:
+
+  staging:
+    email_send_to: 'kernelci-results-staging@groups.io'
```

Comment:
The issue here is that YAML configuration should be about listing the recipients for each combination of kernel tree and report type (build, test etc.).